feature-libs/organization/administration/components/unit/links/users/list/unit-user-link-cell.component.ts

```diff
@@ -32,11 +33,14 @@ export class UnitUserRolesCellComponent extends CellComponent {
   unitKey$: Observable<string> = this.itemService.key$;
   constructor(
     protected outlet: OutletContextData<TableDataOutletContext>,
-    protected itemService: ItemService<B2BUnit>
+    protected itemService: ItemService<B2BUnit>,
+    @Optional() protected b2bUserService: B2BUserService
```

Comment:
Shouldn't we decorate this parameter with a question mark as well?
`protected b2bUserService?: B2BUserService`

Reply:
done